.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "The Lounge",
+  "build": {
+    "dockerfile": "Dockerfile"
+  },
+  "extensions": [
```

Comment:
Only pull in the minimum extensions actually neccessary. Gruntfuggly.todo-tree isn't it.
In fact, you probably shouldn't differ from https://github.com/thelounge/thelounge/blob/master/.vscode/extensions.json overly much.

Reply:
what i had originally but got greedy: https://github.com/thelounge/thelounge/pull/4514/commits/9a11d76d63620048f4a4eaa009859c37d743d9d9